README.md

```diff
@@ -65,6 +131,63 @@ You can also specify the endpoint in the Turing Dev environment:
 MNEMONIC="<MNEMONIC>" ENDPOINT="ws://127.0.0.1:9944" npm run test
 ```
 
+### SDK Tests
+
+You can start the tests by running the following command:
+
+```bash
+MNEMONIC="<MNEMONIC>" ENV="Turing Staging" npm run test:sdk
+```
```

Comment:
Is the `private` folder for wallet secret no longer needed?

Reply:
Yes, use MNEMONIC instead.